### PR TITLE
Some refactoring & all vectorizing.

### PR DESCRIPTION
**Pull request by aulorbe**
Did some minor refactoring (mostly just removing where we write dataframes to files for debugging).

Dropped all cols from `final_master.csv` that we don't need & made the "spoiled" col binary (0 yes, 1 no).

### CODE REVIEW

File: ranked-choice-voting/index.py [hunk 1]
```diff
@@ -17,6 +18,20 @@ def create_dataset_for_modeling(num_ballots_to_generate):
                          sort=False)
 
     master_df = master_df.drop_duplicates()
+
+    # Drop all cols with "Rounds" info:
+    to_filter_out = [col for col in master_df if not col.startswith('Round')]
```
Comment (SherinV) [inline ranked-choice-voting/index.py:23]: `to_filter_out` or `to_keep`?

File: ranked-choice-voting/index.py [hunk 2]
```diff
@@ -17,6 +18,20 @@ def create_dataset_for_modeling(num_ballots_to_generate):
...
+    master_df  = master_df[to_filter_out]
+
+    # Drop other cols we don't :
+    master_df = master_df.drop(columns=['Election', 'filename',
+                                        'pyrankvote_winner_x',
+                                        'condorcet_winner',
+                                        'pyrankvote_winner_y'])
+
+    # Vectorizing dependent var:
+    master_df['spoiled'] = np.where(master_df['spoiled'] == 'Y', 0, 1)  # 0 = yes, 1 = no
```
Comment (SherinV) [inline ranked-choice-voting/index.py:33]: Should it be the other way round? Spoiled? Yes:1 , No:0?

**Review by SherinV — APPROVED** [review]
LGTM

File: ranked-choice-voting/index.py (hunk 2 above)
Comment (Anxhela21) [reply ranked-choice-voting/index.py:33]: yeah, I agree with you Sherin, but we can change later/at any point.

**Pull request merged**